src/gdcdictionary/schemas/aggregated_somatic_mutation.yaml

```diff
@@ -91,6 +91,10 @@ properties:
       - WGS
       - WXS
 
+  platform:
+    $ref:
+      - "read_group.yaml#/properties/platform"
```

Comment:
cool! i didn't know we could do that for non-terms things!

Reply:
Yeah, I feel like we could do this for experimental strategy, but that's for another release.